kong/types.go

```diff
@@ -220,3 +220,16 @@ type Workspace struct {
 	Config    map[string]interface{} `json:"config,omitempty" yaml:"config,omitempty"`
 	Meta      map[string]interface{} `json:"meta,omitempty" yaml:"meta,omitempty"`
 }
+
+// Admin represents an Admin in Kong.
+type Admin struct {
+	CreatedAt        *int    `json:"created_at,omitempty" yaml:"created_at,omitempty"`
+	ID               *string `json:"id,omitempty" yaml:"id,omitempty"`
+	Email            *string `json:"email,omitempty" yaml:"email,omitempty"`
+	Password         *string `json:"password,omitempty" yaml:"password,omitempty"`
```

Comment:
I don't see this field being present in the API response on Kong Enterprise.

Reply:
I feel like I've seen this in the wild but also can't find anything to support that in the docs or tests against an RBAC-enabled instance. I'll remove